xml/System.Threading/CancellationToken.xml

```diff
@@ -510,8 +510,10 @@ For the definition of equality, see the <xref:System.Threading.CancellationToken
 ## Remarks  
  If this token is already in the canceled state, the delegate will be run immediately and synchronously. Any exception the delegate generates will be propagated out of this method call.  
   
- The current <xref:System.Threading.ExecutionContext>, if one exists, will be captured along with the delegate and will be used when executing it.  
-  
+ The current <xref:System.Threading.ExecutionContext> is captured along with the delegate and will be used when executing it. 
+
+ The current <xref:System.Threading.SynchronizationContext> is not captured. When the delegate is executed, the <xref:System.Threading.SynchronizationContext> in use at that time, if one exists, is used. 
```

Comment:
> When the delegate is executed, the <xref:System.Threading.SynchronizationContext> in use at that time, if one exists, is used.

This is misleading and should be removed from both here and the other occurrences below.  When useSynchronizationContext is true, we grab SynchronizationContext.Current, and then the invocation of the delegate is done by passing it to the SynchronizationContext for it to invoke.  If useSynchronizationContext is false, or if an overload is used that doesn't take useSynchronizationContext, then SynchronizationContext doesn't play into the equation at all, so nothing "at that time, if one exists, is used"... no SynchronizationContext is used in that case.